Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -1362,6 +1364,7 @@ def addTruthTrackingGsf(
         "componentMergeMethod": acts.examples.ComponentMergeMethod.maxWeight,
         "mixtureReductionAlgorithm": acts.examples.MixtureReductionAlgorithm.KLDistance,
         "weightCutoff": 1.0e-4,
+        "reverseFilteringCovarianceScaling": 100.0,
```

Comment:
nitpick: should this also be a configurable parameter like for `addKalmanTracks`?

Reply:
let's keep the GSF hardcoded